manifest/wcnspectserv-daemon.yml

```diff
@@ -33,9 +33,7 @@ spec:
       - name: windowswcnspectserver
         image: ghcr.io/microsoft/wcnspect:latest
         command:
-        - powershell.exe
-        - -command
-        - ./wcnspectserv.exe # can add `-p {num}` here to change server's port
+        - /hpc/wcnspectserv.exe # can add `-p {num}` here to change server's port
```

Comment:
ah, This should be here. I was testing in a HPC but I think there is a better way to do this via Working directory